Formula/file-roller.rb

```diff
@@ -22,18 +22,40 @@ class FileRoller < Formula
   depends_on "hicolor-icon-theme"
   depends_on "gnome-icon-theme"
 
+  # Add undeclared dependencies reported by `brew linkage`
+  depends_on "atk"
+  depends_on "cairo"
+  depends_on "gdk-pixbuf"
+  depends_on "gettext"
+  depends_on "glib"
+  depends_on "pango"
+
+  # These can be removed when the next version (>3.16.4) is released, since they
+  # are only required to support the patch below, which will no longer be needed
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "yelp-tools" => :build
+
+  # Remove when >3.16.4 is released
```

Comment:
It would be nice not to modify the DSL, but how would you define the scope of where the relevant part ends? Or just continue with educated guessing?

Also how would you "disable" all version-bound sections en masse to see if it builds without them? That's not strictly necessary but more efficient than manually editing them out one by one just to test. That said, often removing them requires additional changes, so probably less useful than it might seem at first blush.

I'm also not thrilled with copying the literal version string all over the place anyway. Maybe a convention that does `# stable restrict 1.2.3` and `# devel restrict 4.5.6` at the top and then at each affected specific location `# stable restrict` or `# devel restrict`.